src/components/file-list/file-title/file-title.component.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import propTypes from 'prop-types';
+import { Typography } from '@material-ui/core';
+
+import { getFileNameWithoutExtension } from './file-title.utils';
+
+export const FileTitle = ({ title }) => {
+  const isExtensionOn = useSelector((state) =>
```

Comment:
Why the renaming? Try to keep var's naming. This renaming leads to confusion and misreadings.

Reply:
https://github.com/tepachelabs/songbox.webapp/pull/57#discussion_r574951524

We talked about this renaming, we decided to change it. 👀